amazon-kinesis-client/src/test/java/software/amazon/kinesis/application/TestConsumer.java

```diff
@@ -45,8 +48,9 @@
 public class TestConsumer {
     public final KCLAppConfig consumerConfig;
     public final Region region;
-    public final String streamName;
+    public final List<String> streamNames;
     public final KinesisAsyncClient kinesisClient;
```

Comment:
since we've introduced an additional Kinesis client, should we also rename this original client to something more specific for added clarity, i.e. `kinesisClientForConsumerApplication`?